addons/policies/bootstrap_entity_policies.json

```diff
@@ -2221,7 +2223,8 @@
                 [
                     "admin",
                     "service-account-atlan-argo",
-                    "service-account-atlan-backend"
+                    "service-account-atlan-backend",
+                    "atlan-governance-workflows"
```

Comment:
This will allow atlan-governance-workflows user to Attach/detach ANY tag to/from ANY asset in Atlan, just making sure you are aware of it

Reply:
Addition/Removal of Tags is also a requirement under change management workflows. This would be required.